### DISCUSSION TITLE
Reverb

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1733] [General]
Hello, is there a possibility to adjust the reverb stop by stop ?

**Comment by larspalo** [comment]
Depends on what you actually mean by reverb. The only internal reverb that GO has is the Convolution reverb with impulse response files. That is not adjustable on a per rank/pipe basis.

However, if you mean the release tail shortening feature of GO, then that is possible to adjust on both rank and per pipe level. You can easily set this in the Organ Settings dialog on any level of the tree.
